openedx/features/marketplace/views.py

```diff
@@ -12,11 +16,16 @@ def get(self, request, *args, **kwargs):
         return render_to_response(self.template_name, {})
 
 
-class MarketplaceCreateRequestView(View):
+@method_decorator(login_required, name='dispatch')
+class MarketplaceCreateRequestView(generic.CreateView):
```

Comment:
We also need to make sure that the user is affiliated. there must be a decorator lying around for this

Reply:
@saad-waqar affiliated with ??